### PR TITLE
Display moving to ibm-watson

### PR DESCRIPTION
**Pull request by ehdsouza**
This PR displays a warning to a user about the renaming of package

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Python client library to quickly get started with the various [Watson APIs][wdc]
 * You need an [IBM Cloud][ibm-cloud-onboarding] account.
 
 ## Installation
+Note: We are renaming `watson-developer-cloud` to `ibm-watson`
```
Comment (mediumTaj) [inline README.md]: Should this say `renaming`? Maybe we should say something like we are moving to `ibm-watson` and let them know that all versions prior to `v3.0.0` can still be found in `watson-developer-cloud`?

File: watson_developer_cloud/speech_to_text_v1.py [hunk 2]
```diff
@@ -44,7 +44,6 @@
 # Service
 ##############################################################################
 
-
 class SpeechToTextV1(WatsonService):
```
Comment (mediumTaj) [inline watson_developer_cloud/speech_to_text_v1.py]: Should there be deprecation notice here?

File: watson_developer_cloud/text_to_speech_v1.py [hunk 3]
```diff
@@ -40,7 +40,6 @@
 # Service
 ##############################################################################
 
-
 class TextToSpeechV1(WatsonService):
```
Comment (mediumTaj) [inline watson_developer_cloud/text_to_speech_v1.py]: Should there be deprecation notice here?

File: README.md (hunk 1 above)
Comment (ehdsouza) [reply README.md]: ok sounds good

**Pull request merged**